src/meta/test/ActiveHostsManTest.cpp

```diff
@@ -37,6 +37,56 @@ TEST(ActiveHostsManTest, NormalTest) {
     }
 }
 
+TEST(ActiveHostsManTest, NormalTest2) {
```

Comment:
Should give it more meaningful test name like `MergeHostTest` ? WDYT?

Reply:
+1

Reply:
fixed